ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -784,6 +789,9 @@ def wait_until_reboot(self):
             self.do_inboot_oper()
         self.reboot_start = datetime.datetime.now()
         self.log("Dut reboots: reboot start %s" % str(self.reboot_start))
+        if self.kvm_test:
```

Comment:
To keep `wait_until_reboot` simple and generic, can this conditional be moved to `handle_advanced_reboot_health_check_kvm`?

Reply:
Moved the check to separate KVM test from others.